doc/design/abs_backup.md

```diff
@@ -0,0 +1,81 @@
+# Backups using Azure Blob Service (ABS)
+
+In a very similar fashion to backups using [S3](./s3_backup.md), we want to have an option to store backups in ABS.
+
+## Setup
```

Comment:
Remove this section. Just add in the following that it could be configured in cluster spec.